src/NuGet/Microsoft.Net.Compilers.nuspec

```diff
@@ -5,8 +5,7 @@
     <description>
     .Net Compilers package. Referencing this package will cause the project to be built using the specific version of the C# and Visual Basic compilers contained in the package, as opposed to any system installed version.
 
-Supported Platforms:
-- .NET Framework 4.5
+    This package can be used to compile code targeting any platform, but can only be run using the desktop .NET 4.6+ Full Framework.
```

Comment:
"but requires .NET 4.6 to be installed on the developer or build machine"? "can only be run" seems an odd statement for a toolset.